compiler/src/dotty/tools/dotc/transform/SymUtils.scala

```diff
@@ -252,6 +252,9 @@ object SymUtils:
     def isDeprecated(using Context): Boolean =
       self.hasAnnotation(defn.DeprecatedAnnot)
 
+    def isThrows(using Context): Boolean =
```

Comment:
this name implies to me that `self` *is* the annotation, not that it *has* the annotation.
```suggestion
    def hasThrowsAnnot(using Context): Boolean =
```